app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/MapViewModel.kt

```diff
@@ -54,11 +54,10 @@ class MapViewModel constructor(
     }
 
     fun turnOnTreeCaptureMode() {
+        deviceOrientation.enable()
```

Comment:
device orientation is captured and disabled in the image capture activity and hence it has to be enabled for both kasiki hai flow and greenstand flow